std/internal/rpc.js

```diff
@@ -0,0 +1,44 @@
+import { __std } from '__std_generated';
```

Comment:
One comment is that is would be nice to not introduce more `js` here but use `ts`, even if, by nature, there will be a lot of `any`.

Reply:
Yeah no fair enough, ported it to TypeScript